cellrank/tl/kernels/__init__.py

```diff
@@ -2,5 +2,11 @@
 from cellrank.tl.kernels._base_kernel import Kernel, Constant
 from cellrank.tl.kernels._palantir_kernel import PalantirKernel
 from cellrank.tl.kernels._velocity_kernel import VelocityKernel
+from cellrank.tl.kernels._velocity_schemes import (
+    CosineScheme,
+    DotProductScheme,
+    CorrelationScheme,
+    SimilaritySchemeABC,
```

Comment:
I take it this is the base scheme?

Reply:
Yes, it's the abstract base class (ABC) from which users should derive their subclasses.
I didn't want to expose `SimilarityScheme`, because that's what we use (it also includes mandatory hessian function).

Reply:
okay.